test-api/.factorypath

```diff
@@ -0,0 +1,6 @@
+<factorypath>
```

Comment:
This needed?

Reply:
This file is not from me, probably it was added after compiling and is a candidate from .gitignore. Should I add it there?